components/09-nautobot/base/nautobot/templates/configmap.yaml

```diff
@@ -111,3 +111,533 @@ data:
+    #
+    # Example: ALLOWED_HOSTS = ['nautobot.example.com', 'nautobot.internal.local']
+    #
+    # ALLOWED_HOSTS = os.getenv("NAUTOBOT_ALLOWED_HOSTS", "").split(" ")
+
+    # The django-redis cache is used to establish concurrent locks using Redis.
+    #
+    # CACHES = {
+    #     "default": {
+    #         "BACKEND": os.getenv(
+    #             "NAUTOBOT_CACHES_BACKEND",
+    #             "django_prometheus.cache.backends.redis.RedisCache" if METRICS_ENABLED else "django_redis.cache.RedisCache",
+    #         ),
+    #         "LOCATION": parse_redis_connection(redis_database=1),
+    #         "TIMEOUT": 300,
+    #         "OPTIONS": {
+    #             "CLIENT_CLASS": "django_redis.client.DefaultClient",
+    #             "PASSWORD": "",
+    #         },
+    #     }
+    # }
+
+    # Number of seconds to cache ContentType lookups. Set to 0 to disable caching.
+    # CONTENT_TYPE_CACHE_TIMEOUT = int(os.getenv("NAUTOBOT_CONTENT_TYPE_CACHE_TIMEOUT", "0"))
+
+    # Celery broker URL used to tell workers where queues are located
+    #
+    # CELERY_BROKER_URL = os.getenv("NAUTOBOT_CELERY_BROKER_URL", parse_redis_connection(redis_database=0))
+
+    # Database configuration. See the Django documentation for a complete list of available parameters:
+    #   https://docs.djangoproject.com/en/stable/ref/settings/#databases
+    #
+    # DATABASES = {
+    #     "default": {
+    #         "NAME": os.getenv("NAUTOBOT_DB_NAME", "nautobot"),  # Database name
+    #         "USER": os.getenv("NAUTOBOT_DB_USER", ""),  # Database username
+    #         "PASSWORD": os.getenv("NAUTOBOT_DB_PASSWORD", ""),  # Database password
+    #         "HOST": os.getenv("NAUTOBOT_DB_HOST", "localhost"),  # Database server
+    #         "PORT": os.getenv("NAUTOBOT_DB_PORT", ""),  # Database port (leave blank for default)
+    #         "CONN_MAX_AGE": int(os.getenv("NAUTOBOT_DB_TIMEOUT", "300")),  # Database timeout
+    #         "ENGINE": os.getenv(
+    #             "NAUTOBOT_DB_ENGINE",
+    #             "django_prometheus.db.backends.postgresql" if METRICS_ENABLED else "django.db.backends.postgresql",
+    #         ),  # Database driver ("mysql" or "postgresql")
+    #     }
+    # }
+
+    # Ensure proper Unicode handling for MySQL
+    #
+    if DATABASES["default"]["ENGINE"].endswith("mysql"):
+        DATABASES["default"]["OPTIONS"] = {"charset": "utf8mb4"}
+
+    # This key is used for secure generation of random numbers and strings. It must never be exposed outside of this file.
+    # For optimal security, SECRET_KEY should be at least 50 characters in length and contain a mix of letters, numbers, and
+    # symbols. Nautobot will not run without this defined. For more information, see
+    # https://docs.djangoproject.com/en/stable/ref/settings/#std:setting-SECRET_KEY
+    SECRET_KEY = os.getenv("NAUTOBOT_SECRET_KEY", "at)xtt_l9+7uyp-h_@&cc_v_$vw51m*u(9un^oicnxe&_yy8cm")
```

Comment:
We should probably not have a default and require it be passed in no?

Reply:
fair point - it was copy&paste from the existing one, I'll adjust this

Reply:
fixed in the rebase